pysat/instruments/sw_dst.py

```diff
@@ -27,8 +27,12 @@
 """
 
 import datetime as dt
+import ftplib
+from ftplib import FTP
```

Comment:
Another instance of effectively a double import.

Reply:
fixed!